CMakeLists.txt

```diff
@@ -294,12 +294,7 @@ if(NRN_ENABLE_MPI)
   add_definitions("-DOMPI_SKIP_MPICXX=1")
   add_definitions("-DMPICH_SKIP_MPICXX=1")
 
-  # FindMPI broke backward compatibility with v3.19
-  if(${CMAKE_VERSION} VERSION_LESS "3.10")
-    set(NRN_MPIEXEC_EXECUTABLE ${MPIEXEC})
-  else()
-    set(NRN_MPIEXEC_EXECUTABLE ${MPIEXEC_EXECUTABLE})
-  endif()
+  set(NRN_MPIEXEC_EXECUTABLE ${MPIEXEC_EXECUTABLE})
```

Comment:
I think we can drop this line and replace `NRN_MPIEXEC_EXECUTABLE` with `MPIEXEC_EXECUTABLE` elsewhere.

Reply:
Will test that quickly, thanks